platform-dom/src/main/java/com/softicar/platform/dom/elements/input/auto/DomAutoCompleteDisplayStringDeduplicator.java

```diff
@@ -31,54 +30,58 @@
  *
  * @author Oliver Richers
  */
-class DomAutoCompleteDisplayStringDeduplicator<T> {
+public class DomAutoCompleteDisplayStringDeduplicator<T> {
 
 	private final Function<T, IDisplayString> displayFunction;
 	private final Comparator<T> valueComparator;
-	private final Comparator<String> keyComparator;
-	private Map<String, List<T>> listMap;
+	private final DiacriticNormalizer normalizer;
+	private Map<String, List<ValueMapping>> listMap;
 	private Map<String, T> resultMap;
 
 	public DomAutoCompleteDisplayStringDeduplicator(Function<T, IDisplayString> displayFunction, Comparator<T> valueComparator) {
 
 		this.displayFunction = displayFunction;
 		this.valueComparator = valueComparator;
-		this.keyComparator = Comparator.comparing(string -> CurrentDiacriticNormalizationCache.get().normalize(string).toLowerCase());
+		this.normalizer = new DiacriticNormalizer();
```

Comment:
No longer using `CurrentDiacriticNormalizationCache` here will use in a performance regression elsewhere.

Reply:
reverted back to serial stream and using the cache